master/deployment.md

```diff
@@ -19,4 +19,32 @@ $ php artisan route:cache
 $ php artisan migrate --force
 $ php artisan horizon:purge
 $ php artisan storage:link
+$ php artisan horizon:install
+$ php artisan horizon:assets
 ```
+
+### Background Task Runner
+
+Create the file `/etc/systemd/system/pixelfed-queue.service` with the following content:
+
+```
+[Unit]
+Description=Pixelfed task queueing via Laravel Horizon
+After=network.target
+Requires=php7.2-fpm.service
+Requires=redis.service
+Requires=postgresql.service
```

Comment:
postgresql is not officially recommended; mysql or mariadb should take its place here

Reply:
That's right, I'll update it.